### DISCUSSION TITLE
Patched

### DISCUSSION THREAD

**Discussion by nicolasog12** [3kh0/ext-remover#221] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Stopped working, tried every method available. The extension just stays on. Is there any way to work around this

**Comment by nicolasog12** [comment]
I have tried all bookmarklets, direct JS in the search bar, and JS injectors :( it stopped  working

**Comment by Sxri** [comment, marked as answer]
Before making a question or starting a discussion, please read the read me and check other discussions. It has been patched for a while now, its nothing new, chrome patched it in their latest update. However, they're searching for a new method, please be patient
